arch/M68K/M68KDisassembler.c

```diff
@@ -2339,12 +2347,14 @@ static void d68020_extb_32(m68k_info *info)
 
 static void d68000_jmp(m68k_info *info)
 {
+	set_insn_group(info, M68K_GRP_JUMP);
 	cs_m68k* ext = build_init_op(info, M68K_INS_JMP, 1, 0);
```

Comment:
we prefer c89 style. can you move the declaration of `ext` to the beginning of function, before `set_insn_group()` ?

we have this same problem in other places.